### ISSUE TITLE
Bear/Language search bar is broken

### ISSUE THREAD

**Issue by frextrite** [coala/coala#5890]
### Overview Description
Search bar at https://coala.io/#/languages doesn't produce any results while searching for any query

# Steps to Reproduce

1. Go to https://coala.io/#/languages
2. Search for bears/languages in the search bar

# Actual Results
Nothing happens

# Expected Results
Should show relevant results

# Reproducibility

# Additional Information:

**Comment by snow8gaia** [comment]
Can be reproduced.

**Comment by 123vivekr** [comment]
Where should I look into?

**Comment by 123vivekr** [comment]
I would like to fix this issue. Where should I look into?

**Comment by supreet11agrawal** [comment]
@frextrite I want to work upon this issue. Can you tell me the relevant file and assign this to me?

**Comment by subhahu123** [comment]
@frextrite  assign me this issue

**Comment by gurpreetsingh00885** [comment]
I'd like to work on this issue. would be good if someone assigned it to me.

**Comment by frextrite** [comment]
Please ping a maintainer at coala's Gitter channel and ask them to assign this to you.

**Comment by areebbeigh** [comment]
[This issue has already been reported](https://github.com/coala/coala-bears/issues/2632) (and hopefully being worked on).

**Comment by 123vivekr** [comment]
@areebbeigh The issue you referred to is closed and marked as status/invalid

**Comment by areebbeigh** [comment]
Yup and so will this one (probably), I referred to that issue because the discussion has updates on the state of the problem. The last comment on the issue referred above provides links to relevant repos and issues on GitLab.

**Comment by KVGarg** [comment]
@jayvdb @sks444 Running the webservices locally, everything seems fine i.e. all bears are listed and search of a bear is also taking place.
I found that `webservices.coala.io` isn't mentioned in [allowed_hosts settings](https://gitlab.com/coala/landing/blob/master/backend/coala_web/settings.py#L27), maybe due to this we are getting error.
Is there any process or method which can be used to test the changed settings ? I would like to debug this issue.
